src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -5521,10 +5521,60 @@ void CalcAirFlowSimple(EnergyPlusData &state,
             if (thisVentilation.OpenEff != DataGlobalConstants::AutoCalculate) {
                 Cw = thisVentilation.OpenEff;
             } else {
-                // linear interpolation between effective angle and wind direction
-                angle = std::abs(WindDirExt - thisVentilation.EffAngle);
-                if (angle > 180.0) angle -= 180.0;
-                Cw = 0.55 + angle / 180.0 * (0.3 - 0.55);
+                //   Wind Dir (β1)                                        90°, min effectiveness
```

Comment:
I feel like making any commentary on this documentation would make me an accidental art critic.  This is impressive.